### PR TITLE
Project Overview

### PR DESCRIPTION
**Pull request by yuvrajrr**
Check her out, has the project overview minus the component diagram matts working on

### CODE REVIEW

File: docs/Design/SystDesign/SysDes.tex [hunk 1]
```diff
@@ -101,21 +101,33 @@
...
 	\subsection{Normal Behaviour}
-	
+	This application will be used by fitness enthusiasts and athletes alike to track workouts/programs whilst also being able to initeract with worokouts/programs created and posted by other users. 
+
 	\subsection{Undesired Event Handling}
-	
-	\wss{How you will approach undesired events}
-	
+	When an undesired event occurs, the application will ensure it halts and does not progress further. This will prevent users from entering false inputs into the system with the impression that they are acceptable. The appplication will then notify the user of the undesired event and either suggest an option to progress or to return to a previous state of the application. The halting of the application will allow for user data safety to be maintained.
+
```
Comment (bentvelj) [inline docs/Design/SystDesign/SysDes.tex:108]: This section is already under review here: https://github.com/dimitritsampiras/olympian/pull/121

File: docs/Design/SystDesign/SysDes.tex [hunk 2]
```diff
@@ -101,21 +101,33 @@
...
+		The product shall be able to be used by untrained fitness enthusiasts and amateurs alike, who receive no training before using it & UI will be simplistic and consistent throughout, reducing the learning curve for users \\
+		\hline
+		The product shall be usable by users with hearing loss or partial blindness & The application will rely on both audio and visual cues to indicate correct inputs or application events \\
+		\hline
+		The application must inform users when maintenance is taking place and must warn them at least 1 day in advance & The application will utilize pop up screens to display important notifications to the user \\
+		\hline
+		The applicaiton will allow users to report offensive content and remove it from their feed & There will be a button on each post that gives user the option to report offensive content \\
+		\hline
+
+	\end{tabular}
```
Comment (bentvelj) [inline docs/Design/SystDesign/SysDes.tex:130]: For some of these, it seems to just be rewording the NFRs, but I honestly don't really know how you would properly make a single "design decision" to have it understandable by beginners.

Comment (bentvelj) [reply docs/Design/SystDesign/SysDes.tex:130]: So I think this is good

File: docs/Design/SystDesign/SysDes.tex (hunk 1 above)
Comment (yuvrajrr) [reply docs/Design/SystDesign/SysDes.tex:108]: The kanban board lied to me

Comment (bentvelj) [reply docs/Design/SystDesign/SysDes.tex:108]: Yeah the board should definitely have been clearer

**Review by bentvelj — APPROVED** [review]
Good to go

**Pull request merged**